clamav/util.py

```diff
@@ -70,32 +71,47 @@ def scan_stream(fileobj):
     return status, signature_or_none
 
 
-def scan_container_image(image_reference: str):
-    logger.debug(f'scanning container image {image_reference}')
+def yield_files_from_container_image(
```

Comment:
please do not name functions after implementation details. stick with more conventional and abstract names.

For example, `iter` or `enumerate` would be more conventional prefixes. maybe `iter_files`.